gui/src/components/s3/bucket-creation.vue

```diff
@@ -52,7 +52,7 @@
                   <v-tooltip right max-width="300" v-if="props.item.bucket_url">
                     <template v-slot:activator="{ on }">
                       <img
-                        id="s3-edit-account"
+                        :id="'copy-bucket-url-'+props.item.name"
```

Comment:
Can we please use string interpolation here instead of string concatenation ?

Reply:
@namrata-khake-ui  Please take action

Reply:
Updated review comments